.github/workflows/optimize-ci.yml

```diff
@@ -60,6 +60,7 @@ jobs:
 
   es-schema-integrity-test:
     name: Elasticsearch Schema Integrity Test
+    if: false
```

Comment:
Are we setting this to false because we have no 8.7?

Reply:
Yes, that is the intention and then we would readd it after the 8.7 release. I was experimenting a little with it here though, this PR wasn't strictly ready for review (thanks for the early feedback anyway though 🙂)

Reply:
Will be readded in https://github.com/camunda/camunda/issues/27314